resourcifier/Makefile

```diff
@@ -34,5 +40,8 @@ endif
 container:
 	docker build -t $(PREFIX)/$(IMAGE):$(TAG) -f Dockerfile $(DIR)
 
+container-push:
```

Comment:
Probably better to call this docker-push to make clear how it differs from push.